### DISCUSSION TITLE
WELCOME TO OUR GITHUB GROUP

### DISCUSSION THREAD

**Discussion by LimitedJert** [LimitedJert/ENG220-Group13#1] [Announcements]
*Hello, Members from the UNM Business & Peace Engineering*

This would be our main work flow for doing adjustment to code or 
whatever task has been given to us.

This is my first time doing github but I have in the past done other thing like
Game modifying and server running are done using GitHub communities.

**Comment by SebastianAbeyta** [comment]
Hello, this is Sebastian Abetya!
